### PR TITLE
refactor(style): rename style_anim_time -> style_anim_duration

### PR DESCRIPTION
**Pull request by kisvegabor**
Help us review this PR! Anyone can [approve it or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### Description of the feature or fix

Anim "time" was already renamed to "duration" on other places. 

### Checkpoints
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html)

### CODE REVIEW

**Comment by kisvegabor** [comment]
cc @XuNeo

File: examples/widgets/list/lv_example_list_1.c [hunk 1]
```diff
@@ -21,7 +21,7 @@ void lv_example_list_1(void)
     /*Add buttons to the list*/
     lv_obj_t * btn;
     lv_list_add_text(list1, "File");
-    btn = lv_list_add_button(list1, LV_SYMBOL_FILE, "New");
+    btn = lv_list_add_button(list1, LV_SYMBOL_FILE, "New    aaaaaaaaaaaaaaaaaa      ssssssdd ddddddd");
```
Comment (IAMMX) [inline examples/widgets/list/lv_example_list_1.c]: why add this code？

Comment (XuNeo) [inline examples/widgets/list/lv_example_list_1.c]: Need to remove.

**Review by XuNeo — COMMENTED** [review]
Thank you Gabor. Looks good to me.
You may also want to add changes to lv_api_map.h
`LV_STYLE_ANIM_TIME ` and `lv_obj_set/get_style_anim_time`

File: examples/widgets/list/lv_example_list_1.c (hunk 1 above)
Comment (kisvegabor) [reply examples/widgets/list/lv_example_list_1.c]: Upps, test code. Removed :slightly_smiling_face:

**Comment by kisvegabor** [comment]
Updated. The `LV_STYLE_...` properties are not added to `lv_api_map.h`, but I added the related functions.

**Comment by XuNeo** [comment]
> The `LV_STYLE_...` properties are not added to `lv_api_map.h`

I think a simple `define LV_STYLE_ANIM_TIME LV_STYLE_ANIM_DURATION` could work. No?

**Comment by kisvegabor** [comment]
I added all `LV_STYLE...`s.

**Comment by kisvegabor** [comment]
Bypassing the branch protection for the release candidate today.

**Pull request merged**